starter-code/index.html

```diff
@@ -1,31 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Instagram</title>
-</head>
-<body>
-  Instagram
+  <head>
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Instagram</title>
+  </head>
+  <body>
+    <main class="main">
+      <aside class="aside">
+        <img class="aside-img" src="./images/phones.png" alt="Phones" />
+      </aside>
 
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
-</body>
+      <section class="section">
+        <div class="top-content">
+          <img src="./images/brand.png" alt="instagram-brand-icon" />
+          <button class="login">
+            <img class="top-img" src="./images/home.png" alt="home-icon" />
+            Log in
+          </button>
+        </div>
+        <div class="container">
+          <div class="content">
+            <h1 class="title">Capture and Share the World's Moments</h1>
+            <p>
+              Instagram is a <b>fast</b>,<b> beautiful</b> and <b>fun</b> way to
+              share your life with friends and family.
+            </p>
+            <p>
+              Take a picture or video, choose a filter to transform its look and
+              feel, then post to Instagram &mdash; it's that easy. You can even
+              share to Facebook, Twitter, Tumblr and more. It's a new way to see
+              the world.
+            </p>
+            <p>Oh yeah, did we mention it's free?</p>
+            <div class="badge">
+              <img
+                class="badge-img"
+                src="./images/badge-iphone.png"
+                alt="Apple-App"
+              />
+              <img
+                class="badge-img"
+                src="./images/badge-android.png"
+                alt="Android-App"
+              />
+            </div>
+          </div>
+        </div>
+      </section>
+      <footer class="footer">
+        <ul class="footer-content">
+          <li><a href=""> About Us </a></li>
```

Comment:
Here, if you should add a `#` in the href if you don't know the link yet. It stops the page from refreshing when you click on it